content/2018-06-11_pyconcz-2018-report.md

```diff
@@ -0,0 +1,104 @@
+Title:  PyConCz 2018 — Report
+Date: 2018-06-11 00:00:00
+Author: Josef Pospíšil
+About: Visit [Josef's blog](https://medium.com/@damnpepe) on Medium.
+
+> This article first appeared on Josef's Medium blog
```

Comment:
This note is the thing that get's visible on the fornt page. Can we maybe move it after the first paragraph?

Reply:
Or maybe at the end?